Allow docs build without already installed libgxps (gtk-doc scan generates <libgxps/gxps.h>)
--- libgxps-0.2.1/docs/reference/Makefile.am.orig	2011-10-08 14:16:05.000000000 +0200
+++ libgxps-0.2.1/docs/reference/Makefile.am	2012-01-28 14:23:48.724403590 +0100
@@ -68,8 +68,8 @@
 # e.g. INCLUDES=-I$(top_srcdir) -I$(top_builddir) $(GTK_DEBUG_FLAGS)
 # e.g. GTKDOC_LIBS=$(top_builddir)/gtk/$(gtktargetlib)
 INCLUDES=				\
-	-I$(top_srcdir)/libgxps		\
-	-I$(top_builddir)/libgxps	\
+	-I$(top_srcdir)		\
+	-I$(top_builddir)	\
 	$(GXPS_CFLAGS)
 
 GTKDOC_LIBS=					\
